tools/Config.mk

```diff
@@ -608,6 +620,42 @@ define DOWNLOAD
 	$(ECHO_END)
 endef
 
+# CLONE - Git clone repository. Initializes a new Git repository in the 
+#         folder on your local machine and populates it with the contents
+#         of the central repository.
+#         The third argument is an storage path. The second argument is used
+#         if it is not provided or is empty.
+# Example: $(call CLONE,$(URL_BASE),$(PATH),$(STORAGE_FOLDER))
+
+define CLONE
+	$(ECHO_BEGIN)"Clone: $(if $3,$3,$2) "
+	if [ -z $3 ]; then \
+		git clone --quiet $1 $2; \
+	else \
+		if [ ! -d $3 ]; then \
+			git clone --quiet $1 $3; \
+		fi; \
+		cp -fr $3 $2; \
+	fi
```

Comment:
Same here: we can add more verbosity for the cases where we provide the storage folder and copy from the storage folder to the final path.